spec/spec_helper.rb

```diff
@@ -149,3 +149,10 @@ def env_args_to_hash(*args)
     Jekyll::GitHubMetadata.logger = Logger.new(StringIO.new) unless ENV["DEBUG"]
   end
 end
+
+def versions
+  require "github-pages"
+  GitHubPages.versions
+rescue LoadError
+  {}
+end
```

Comment:
Rubocop complained that it didn't like the original chained call, as it was sandwiched in a lamda inside a hash which ended with `end.call`. This was my best attempt to preserve the original behavior while not angering Rubocop.